### PR TITLE
feat: add core sdk kotlin jvm docs

### PR DESCRIPTION
**Pull request by dom-murray**
## Description

Adds a page for the newly available Immutable X Core SDK Kotlin/JVM under the SDKs section. 

## Resolved issues
Jira:

### Before submitting the PR, please consider the follow:
- [x] It's beneficial if your pull request references an issue used to discuss the pull request ahead of time. If you haven't previously created an issue, please create one and discuss your contribution with the maintainers.
- [x] Prefix your PR title with `feat: `, `fix: `, `chore: `, `docs:`, or `refactor:`.
- [ ] The description should clearly illustrate what problems the pull request solves.
- [x] Ensure that the commit messages follow our guidelines.
- [x] Resolve merge conflicts (if any).
- [x] Make sure that the current branch is up to date with the `main` branch.

### CODE REVIEW

File: docs/sdks/Core SDK Kotlin JVM.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+```kt
+val response = CollectionsApi().getCollections(
+    pageSize = 20,
+    orderBy = "name",
+    direction = "asc"
+)
+```
+View the [OpenAPI spec](https://github.com/immutable/imx-core-sdk-kotlin-jvm/blob/main/openapi.json) for a full list of API requests available in the Core SDK.
+
+## Workflow Functions
```
Comment (sprngbk) [inline docs/sdks/Core SDK Kotlin JVM.md:57]: Would a table like this in the [Typescript SDK](https://docs.x.immutable.com/docs/imx-core-sdk-ts#available-workflows) be clearer for users?

File: docs/sdks/Core SDK Kotlin JVM.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+e.g. Get a list of collections ordered by name in ascending order
+```kt
+val response = CollectionsApi().getCollections(
+    pageSize = 20,
+    orderBy = "name",
+    direction = "asc"
+)
+```
+View the [OpenAPI spec](https://github.com/immutable/imx-core-sdk-kotlin-jvm/blob/main/openapi.json) for a full list of API requests available in the Core SDK.
```
Comment (sprngbk) [inline docs/sdks/Core SDK Kotlin JVM.md:55]: Would it be worth linking to Dokka docs here as well?

**Review by sprngbk — COMMENTED** [review]
Thanks @dom-murray. Just a couple of suggestions. Let me know what you think!

File: docs/sdks/Core SDK Kotlin JVM.md (hunk 2 above)
Comment (sprngbk) [reply docs/sdks/Core SDK Kotlin JVM.md:55]: Will do in another ticket

Comment (dom-murray) [reply docs/sdks/Core SDK Kotlin JVM.md:55]: We removed the Dokka docs before but we can re-generate and add them in. I'll do it in another PR when it's ready.

**Review by sprngbk — APPROVED** [review]

**Pull request merged**